lib/models/device.js

```diff
@@ -2711,6 +2711,79 @@ const getDeviceModel = function (deps, opts) {
 			});
 		},
 
+		/**
+		 * @summary Set a specific device to run a particular supervisor release
+		 * @name setSupervisorRelease
+		 * @public
+		 * @function
+		 * @memberof balena.models.device
+		 *
+		 * @description Configures the device to run a particular supervisor release.
+		 *
+		 * @param {String|Number} uuidOrId - device uuid (string) or id (number)
+		 * @param {String|Number} supervisorVersionOrId - the version of a released supervisor (string) or id (number)
```

Comment:
This is an interesting case. `null` is kind of special due to how legacy devices provision. I _think_ that's the only time it makes sense to allow setting to `null`, otherwise we will wind up allowing implicit downgrades like so: `version B` -> `null` -> `version A` such that B > A (see https://github.com/balena-io/open-balena-api/commit/68120045cf898cb068d361256ad65a30372658a6). Given that, I think the answer is no, we don't want to allow that, but can't tighten up the rule at the moment.

cc @CameronDiver

Reply:
Actually, I wonder if we should specifically disable that codepath at the API level as well...